utility/src/main/java/org/oppia/android/util/parser/ImageLoader.kt

```diff
@@ -1,21 +1,12 @@
 package org.oppia.android.util.parser
 
-import android.graphics.Bitmap
-import android.graphics.drawable.PictureDrawable
-
 /** Loads an image from the provided URL into the specified target, optionally caching it. */
 interface ImageLoader {
-  /**
-   * Loads a bitmap at the specified [imageUrl] into the specified [target]. Note that this is an
-   * asynchronous operation, and may take a while if the image needs to be downloaded from the
-   * internet.
-   */
-  fun loadBitmap(imageUrl: String, target: ImageTarget<Bitmap>)
 
   /**
-   * Loads a vector drawable at the specified [imageUrl] into the specified [target]. Note that this
-   * is an asynchronous operation, and may take a while if the image needs to be downloaded from the
+   * Loads an oppiaImage at the specified [imageUrl] into the specified [target]. Note that this is an
+   * asynchronous operation, and may take a while if the image needs to be downloaded from the
    * internet.
    */
-  fun loadSvg(imageUrl: String, target: ImageTarget<PictureDrawable>)
+  fun loadOppiaImage(imageUrl: String, target: ImageTarget<OppiaImage>)
```

Comment:
I think it's okay to have both loadBitmap and loadSvg--we probably shouldn't assume a URL ends with svg. This is a not-so-great assumption by ``UrlImageParser``, but better to do it at the HTML level than in the actual image loader.

Reply:
Okay, reverted these changes